cmd/up/up_test.go

```diff
@@ -307,3 +308,47 @@ func TestEnvVarIsNotAddedWhenHasBuiltInOktetoEnvVarsFormat(t *testing.T) {
 		})
 	}
 }
+
+func TestCommandAddedToUpOptionsWhenPassedAsArgument(t *testing.T) {
+	var tests = []struct {
+		name            string
+		upOptions       *UpOptions
+		args            []string
+		Manifest        *model.Manifest
+		expectedCommand []string
+	}{
+		{
+			name:            "Passing dev environment but no command",
+			upOptions:       &UpOptions{},
+			args:            []string{},
+			Manifest:        &model.Manifest{Dev: map[string]*model.Dev{"frontend": {}}},
+			expectedCommand: nil,
+		},
+		{
+			name:            "Passing command but no dev environment",
+			upOptions:       &UpOptions{},
+			args:            []string{"echo", "hello"},
+			Manifest:        &model.Manifest{Dev: map[string]*model.Dev{}},
+			expectedCommand: []string{"echo", "hello"},
+		},
+		{
+			name:            "Passing command and dev environment",
+			upOptions:       &UpOptions{},
+			args:            []string{"echo", "hello"},
+			Manifest:        &model.Manifest{Dev: map[string]*model.Dev{"frontend": {}}},
+			expectedCommand: []string{"echo", "hello"},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			err := tt.upOptions.AddArgs(tt.args, tt.Manifest)
+			if err != nil {
```

Comment:
@abs007 We usually use [testify](https://github.com/stretchr/testify) to test, since it's more readable
```suggestion
			assert.NoError(t, err)
```

Reply:
Should I instead go with `if assert.Error(t, err) ` over here?